packages/react-components/react-tabs/src/components/TabList/TabList.types.ts

```diff
@@ -47,6 +47,15 @@ export type TabListProps = ComponentProps<TabListSlots> & {
    */
   appearance?: 'transparent' | 'subtle';
 
+  /**
+   * Tab size may change between unselected and selected states.
+   * The default scenario is a selected tab has bold text.
+   *
+   * When true, this property requests tabs be the same size whether unselected or selected.
+   * @default 'true'
```

Comment:
default true is typically an antipattern. is there a specific need to do it this way?

Reply:
A variable named in the the negative is generally regarded as confusing to callers (e.g. disableReserveSelectedTabSpace).  The default case here is to have the built-in Tab not pixel jump between unselected and selected states, so it seems the best approach to have the feature on by default.